core/server/api/schedules.js

```diff
@@ -6,7 +6,7 @@ const Promise = require('bluebird'),
     models = require('../models'),
     config = require('../config'),
     common = require('../lib/common'),
-    postsAPI = require('../api/posts');
+    postsAPI = require('./posts');
```

Comment:
maybe this doesn't count because it's _inside_ the api? but this could be `api = require('../api')` and then usage below for `api.posts`

Reply:
I am pretty sure you will get a circular dependency error.
Because api requires schedules, schedules requires api. This was the main reason for doing

> postsAPI = require('../api/posts');